### PR TITLE
Feature/placeholder prop inputs

### PR DESCRIPTION
**Pull request by LEduard0**
#### What problem is this solving?

Currently, the store-form component does not have an option to add a placeholder to inputs

#### How to test it?

1. Enter the workspace and scroll until reach the form
2. According to the value passed in the placeholderText property, the placeholder will be displayed in the input

[Workspace](https://will--mundopet.myvtex.com/osso-mastig-natural-femur-suino-para-caes/p?skuId=757)

#### Screenshots or example usage:

https://prnt.sc/vhk3fr

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -147,13 +147,20 @@ In the example below, the form block is contained in a Flex Layout row:
...
 | `label` | `string` |  ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Field's name when rendered | Property's title  |
 
+### `form-input.textarea` props
+
+| Prop name   | Type                                 | Description                                                                                                                                                                                                                                      | Default Value |
+| ----------- | ----------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| `placeholderText`   | `string`    |  Placeholder text for the textarea input.	 | `undefined`              |
```
Comment (klzns) [inline docs/README.md]:
```suggestion
| `placeholder`   | `string`    |  Placeholder for the textarea input.	 | `undefined`              |
```

File: docs/README.md [hunk 2]
```diff
@@ -147,13 +147,20 @@ In the example below, the form block is contained in a Flex Layout row:
...
 | `pointer`   | `string`    | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) JSON schema pointer i.e. the JSON schema path  (for example: #/properties/firstName) in which the form block inputs should be validated against. | `undefined`              |
 | `inputType` | `enum` | Defines which type of a text field should be rendered: <br>`input`: renders a normal text field.<br>`hidden`: does not render any text field. It should be used in scenarios in which you want to pre-define a field value to be submitted to the form but that shouldn't be visible (and therefore editable) to users. <br>`password`: renders a password text field.                 | `input`           |
 | `label` | `string` |  ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) Field's name when rendered | Property's title  |
+| `placeholderText`   | `string`    |  Placeholder text for the text input.	 | `undefined`              |
```
Comment (klzns) [inline docs/README.md]:
```suggestion
| `placeholder`   | `string`    |  Placeholder for the text input.	 | `undefined`              |
```

File: react/components/Input.tsx [hunk 3]
```diff
@@ -23,17 +25,25 @@ export const PasswordInput: FC<FormRawInputProps> = props => {
 }
 
 export const RawInput: FC<FormRawInputProps> = props => {
-  const { pointer, label } = props
+  const { pointer, label, placeholderText } = props
```
Comment (klzns) [inline react/components/Input.tsx]:
```suggestion
  const { pointer, label, placeholder } = props
```

File: react/components/Input.tsx [hunk 4]
```diff
@@ -23,17 +25,25 @@ export const PasswordInput: FC<FormRawInputProps> = props => {
...
+      label={label}
+      placeholderText={placeholderText}
+    />
+  )
 }
 
 export const Input: FC<{
   inputObject: UseRawInputReturnType
   label?: string
+  placeholderText?: string
```
Comment (klzns) [inline react/components/Input.tsx]:
```suggestion
  placeholder?: string
```

File: react/components/Input.tsx [hunk 5]
```diff
@@ -23,17 +25,25 @@ export const PasswordInput: FC<FormRawInputProps> = props => {
...
 export const RawInput: FC<FormRawInputProps> = props => {
-  const { pointer, label } = props
+  const { pointer, label, placeholderText } = props
   const inputObject = useInput(pointer)
-  return <Input inputObject={inputObject} label={label} />
+  return (
+    <Input
+      inputObject={inputObject}
+      label={label}
+      placeholderText={placeholderText}
```
Comment (klzns) [inline react/components/Input.tsx]:
```suggestion
      placeholder={placeholder}
```

**Comment by klzns** [comment]
Thanks for the PR!

I changed the prop name to `placeholder`.

I tested in this workspace (vtex only): https://breno--storecomponents.myvtex.com/contact-us

Changed I did to the store-theme:
image

**Review by klzns — APPROVED** [review]

**Review by icazevedo — APPROVED** [review]

**Review by guerreirobeatriz — APPROVED** [review]

**Review by kaisermann — APPROVED** [review]

**Comment by klzns** [comment]
@allcontriburtors add @LEduard0 for code

**Comment by klzns** [comment]
@allcontributors add @LEduard0 for code

**Pull request merged**